velox/dwio/parquet/writer/Writer.h

```diff
@@ -109,6 +109,8 @@ struct WriterOptions : public dwio::common::WriterOptions {
   /// Timestamp unit for Parquet write through Arrow bridge.
   /// Default if not specified: TimestampUnit::kNano (9).
   std::optional<TimestampUnit> parquetWriteTimestampUnit;
+  /// Timestamp time zone for Parquet write through Arrow bridge.
+  std::optional<std::string> parquetWriteTimestampTimeZone{std::nullopt};
```

Comment:
I suppose you don't need to initialize by `std::nullopt`, `std::optional<std::string> parquetWriteTimestampTimeZone`  is enough.

Reply:
@jinchengchenghh Yes. Updated.